assets/pages/support-landing/components/contributionPaymentCtasContainer.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { connect } from 'react-redux';
+
+import ContributionPaymentCtas from 'components/contributionPaymentCtas/contributionPaymentCtas';
+import {
+  getAmount,
+} from 'components/contributionSelection/contributionSelectionReducer';
+import {
+  payPalContributionButtonActionsFor,
+} from 'components/paymentButtons/payPalContributionButton/payPalContributionButtonActions';
+
+import type { State } from '../supportLandingReducer';
+
+
+// ----- State Maps ----- //
+
+function mapStateToProps(state: State) {
```

Comment:
Is it possible to type the return object of this function? It happened to me before that I didn't set up the `countryGroup` and that silently failed in another part of the site.

Reply:
The error I mentioned above is [here](https://github.com/guardian/support-frontend/pull/504/commits/cd8f87641ef16f935cad89bb4b783125f951ae20#diff-97cce600f7d93428c022d02ff743a0acR77).

Reply:
We don't apply return types to this function anywhere in the codebase, I think probably because it's a simple object mapping, so the type would just look identical to the returned object. In this situation I think you're probably as likely to forget to update the type as you are the object.